docs/SUMMARY.md

```diff
@@ -75,6 +75,7 @@
     * [Freshdesk](integrations/sources/freshdesk.md)
     * [Freshsales](integrations/sources/freshsales.md)
     * [Freshservice](integrations/sources/freshservice.md)
+    * [Gas Prices](integrations/sources/gas-prices.md)
```

Comment:
The connector name should be USA Gas Prices because is only possible to get data from US states.